SUSE/connect-ng
# Correct CPU information gathering on ppc64

## How to test

1. Run `make build-ppc64le`.
2. Upload the resulting binary into a PowerPC machine (ask me for access to one).
3. Run SUSEConnect with `--debug` and see new parameters like `device_tree` or `lpar_mode` being sent inside of `arch_specs`.
